exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+
+  <div id="perseus">
+    <div>
+      <div id="problem-area">
+        <div id="workarea"></div>
+        <div id="hintsarea"></div>
+      </div>
+      <div id="answer-area-wrap">
+          <div id="answer-area">
+              <div class="info-box">
+                  <div id="solutionarea"></div>
+                  <button @click="checkAnswer" v-if="!correct" id="check-answer-button">{{ checkText }}</button>
+                  <button v-else id="next-question-button">{{ $tr('correct') }}</button>
+              </div>
+              <div class="info-box">
+                  <button @click="takeHint">
+                    {{ $tr("hint") }}
+                  </button>
+              </div>
+          </div>
+      </div>
+      <div style="clear: both;"></div>
+    </div>
+    <div id="scratchpad-btn-container">
+        <button v-if="scratchpad" id="scratchpad-show">{{ $tr("showScratch") }}</button>
+        <button v-else disabled id="scratchpad-not-available">{{ $tr("notAvailable") }}</button>
+    </div>
+    <div v-el:perseus-container id="perseus-container">
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    init() {
+      // Perseus expects React to be available on the global object
+      this.react = global.React = require('react');
```

Comment:
did some search but I cannot find the answer on who introduced this `global.xxx` way of expose object to the global scope